core/templates/components/skill-mastery/skill-mastery.component.ts

```diff
@@ -0,0 +1,84 @@
+// Copyright 2019 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the skill mastery viewer.
+ */
+
+import { SkillMasteryListConstants } from
+  'components/skills-mastery-list/skills-mastery-list.constants';
+
+import { Component, Input, OnInit } from '@angular/core';
+import { downgradeComponent } from '@angular/upgrade/static';
+
+import { SkillMasteryBackendApiService } from
+  'domain/skill/skill-mastery-backend-api.service';
```

Comment:
```suggestion
import { Component, Input, OnInit } from '@angular/core';
import { downgradeComponent } from '@angular/upgrade/static';

import { SkillMasteryListConstants } from
  'components/skills-mastery-list/skills-mastery-list.constants';
import { SkillMasteryBackendApiService } from
  'domain/skill/skill-mastery-backend-api.service';
```

The import order we follow is:

1. First party imports (mostly imports from @angular)
2. Third-party imports (imports from other libraries)
3. Application imports (imports from within our application)

Within these subcategories, it is sorted by where the module is imported from:
So the right way to import is:
```
import { D } from 'A';
import { B } from 'C';
```
because 'A' < 'C'.

Reply:
Got it! Done